bucket/logseq-nightly.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "0.5.8-20220108",
+    "description": "A privacy-first platform for knowledge sharing and management",
+    "homepage": "https://logseq.com",
+    "license": "AGPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/logseq/logseq/releases/download/nightly/logseq-win-x64-0.5.8+nightly.20220108.exe#/dl.7z",
+            "hash": "15ee81d2e76b40c03b86364ac35fbc1831bb78ac22246447572d4eafad9ca7d3"
+        }
+    },
+    "pre_install": [
+        "Expand-7ZipArchive \"$dir\\Logseq-*-full.nupkg\" -ExtractDir 'lib\\net45' -Removal",
+        "Remove-Item \"$dir\\lib\", \"$dir\\Update*\", \"$dir\\*.gif\", \"$dir\\*.ico\", \"$dir\\RELEASES*\" -Recurse"
+    ],
+    "shortcuts": [
+        [
+            "Logseq.exe",
+            "Logseq"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/logseq/logseq/releases/tag/nightly",
+        "regex": "Logseq-win-x64-(?<main>[\\d.]+)\\+nightly\\.(?<date>\\d{8})",
+        "replace": "${main}-${date}"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/logseq/logseq/releases/download/nightly/logseq-win-x64-$matchMain+nightly.$matchDate.exe#/dl.7z",
+                "hash": {
+                    "url": "$baseurl/SHA256SUMS.txt",
+                    "regex": "$sha256\\s{2}Logseq-win-x64-$matchMain+nightly\.$matchDate\.exe"
```

```suggestion
                    "regex": "$sha256\\s{2}Logseq-win-x64-$matchMain+nightly\\.$matchDate\\.exe"
```

Sorry, my bad. Two slashes each.

---

Haha, I just wanted to correct thanks XD